packages/neuron-wallet/src/services/cells.ts

```diff
@@ -1054,4 +1055,39 @@ export default class CellsService {
 
     return outputEntities
   }
+
+  public static async getMultisigBalance(isMainnet: boolean) {
```

Comment:
If `args` list is not specified in the query, we should add extra logic to guarantee cells of removed addresses are removed at the same time. Or the query may be consuming.

Reply:
I want't to get all live multisig address balance.Even if the addresses has removed,I can also add it again.It's slowly if search all live multisig addresses?

Reply:
Without cleaning, the data will accumulate by adding new addresses. It would be disturbing to users who often approve transactions.

Reply:
ok, I will add added addresses list as a parameter